src/components/policy/src/policy/src/policy_manager_impl.cc

```diff
@@ -316,16 +316,10 @@ bool PolicyManagerImpl::ResetUserConsent() {
   return result;
 }
 
-void PolicyManagerImpl::SendNotificationOnPermissionsUpdated(
```

Comment:
Isn't that too much of changes in SDL's code? Could we keep old method and add new one?